sdk/purview/azure-analytics-purview-sharing/README.md

```diff
@@ -0,0 +1,346 @@
+# Azure Purview Sharing client library for Java
+
+**Please rely heavily on the [service's documentation][product_documentation] and our [data-plane client docs][protocol_method] to use this library**
```

Comment:
dont see a hyperlink for [protocol_method]

Reply:
Good catch!  Removed this.